shapeshifter/mixins.py

```diff
@@ -0,0 +1,15 @@
+from django.contrib import messages
+
+
+class MultiSuccessMessageMixin(object):
+    """
+    Add a success message on successful multiple forms submission.
+    """
+    success_message = None
+
+    def forms_valid(self):
+        response = super(MultiSuccessMessageMixin, self).forms_valid()
+        if self.success_message:

```

Comment:
Probably need to follow something similar to [how we do it `braces`](https://github.com/brack3t/django-braces/blob/master/braces/views/_forms.py#L124) since we could get something not-exactly-text or that needs to be processed in some way.

At the least have a `get_success_message` method?